doc/classes/MultiplayerPeer.xml

```diff
@@ -12,6 +12,13 @@
 		<link title="WebRTC Signaling Demo">https://godotengine.org/asset-library/asset/537</link>
 	</tutorials>
 	<methods>
+		<method name="generate_unique_id" qualifiers="const">
+			<return type="int">
+			</return>
+			<description>
+				Returns a randomly generated integer that can be used as a network unique ID.
```

Comment:
Is this ID persistent across restarts? This should be documented.

Reply:
> Is this ID persistent across restarts or client reconnections?

No, there is no guarantee about that, it's a pseudo-random number as mentioned.